cmd/kubeadm/app/phases/addons/dns/versions.go

```diff
@@ -23,13 +23,13 @@ import (
 
 const (
 	kubeDNSv190AndAboveVersion = "1.14.7"
-	coreDNSVersion             = "1.0.1"
+	coreDNSVersion             = "1.0.2"
```

Comment:
Do we want to update the version in kubeadm each time the new CoreDNS version is released? @luxas

Reply:
But yeah it'd be ok if the new CoreDNS version doesn't break anything of kubernetes.

Reply:
Good point, the 1.0.2 release notes categorize 1.0.2 as focused on facilitating external plugin development. Kubernetes (a native plugin), is not mentioned.  So, there appears to be no functional need to go to 1.0.2.  The deployment script at coredns/deployment is still at 1.0.1 at the moment.

Reply:
I did not see it.
But I think we should stick to initial plan : we report for deployment tools (kubeadm, kube-up etc ...) the same manifest - and then image, that is used in our coredns/deployment/kubernetes repo.

which means keep for now on 1.0.1. (else we may reproduce the initial error that was to deploy a manisfest that is not in line with the image version)